CHANGELOG.md

```diff
@@ -37,6 +38,7 @@ should change the heading of the (upcoming) version to include a major version b
 ## Dev / docs / playground
 
 - Updated docs and playground with the implementation guide of newly added date re-order feature.
+- [4024](https://github.com/rjsf-team/react-jsonschema-form/issues/4024) Updated the base64 references from (`atob` and `btoa`) to invoke the functions from the new `base64` object in `@rjsf/utils`.
```

Comment:
Can you also update the utils API documentation located here `packages/docs/docs/api-reference/utility-functions.md`

Reply:
updated